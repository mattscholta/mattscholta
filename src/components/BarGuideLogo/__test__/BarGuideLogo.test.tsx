import * as React from 'react';
import { render, RenderResult } from '@testing-library/react';

import { BarGuideLogo, BarGuideLogoProps } from '../BarGuideLogo';

describe('BarGuideLogo', () => {
  let props: BarGuideLogoProps;
  let wrapper: RenderResult;

  const createWrapper = () => <BarGuideLogo {...props} />;

  beforeEach(() => {
    props = {};
    wrapper = render(createWrapper());
  });

  it('matches snapshot', () => {
    expect(wrapper.container.childNodes).toMatchSnapshot();
  });
});
