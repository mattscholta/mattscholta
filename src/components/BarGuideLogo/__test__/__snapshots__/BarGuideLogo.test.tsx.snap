// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BarGuideLogo matches snapshot 1`] = `
NodeList [
  <svg
    class="component"
    fill="none"
    height="50"
    viewBox="0 0 51 51"
    width="50"
  >
    <path
      d="M40.2253 8.31595L38.8888 4.18905L34.5664 4.73994L32.0686 1.20428L28.0862 3.00337L24.6491 0.373108L21.3605 3.26055L17.2896 1.76939L14.9869 5.48861L10.6438 5.26906L9.53173 9.48959L5.30238 10.5611L5.47972 14.908L1.7399 17.1754L3.1909 21.2623L0.272919 24.5242L2.86864 27.988L1.03177 31.9545L4.54158 34.4875L3.94903 38.8061L8.06106 40.1832L8.76549 44.4702L11.4655 44.5316C11.4103 44.0623 11.4489 43.5737 11.6006 43.0733C13.6207 35.8445 14.5664 28.8238 15.0503 23.4654C15.4106 19.4764 15.6923 16.358 16.412 14.6348C19.1972 7.96442 40.7765 17.0441 37.9943 23.7155C37.2766 25.4358 35.2654 27.8202 32.6912 30.872C32.2035 31.4502 31.6937 32.0318 31.1662 32.6335L31.1655 32.6343L31.1647 32.6352C28.2262 35.9872 24.7417 39.962 21.4932 47.4388C21.2282 47.8963 20.884 48.2583 20.4831 48.5339L22.2534 50.3731L25.93 48.0034L29.7264 50.0874L32.5537 46.7467L36.8082 47.6117L38.5349 43.597L42.8695 43.1662L43.3421 38.8341L47.3717 37.1458L46.5484 32.8813L49.9149 30.0854L47.8686 26.2674L50.2729 22.6124L47.1857 19.5801L48.4141 15.3907L44.5601 13.4137L44.5036 9.06221L40.2253 8.31595Z"
      fill="#FF0000"
    />
    <path
      d="M27.574 20.3014C28.7645 20.8031 29.6965 20.6442 30.5103 20.5054C31.3253 20.3664 32.0217 20.2476 32.7404 20.8332C35.0575 22.7389 29.9486 30.1498 27.7828 29.2381C27.1395 28.9685 27.1929 28.6424 27.2494 28.2974C27.305 27.9581 27.3636 27.6006 26.7658 27.2605C25.2151 26.4302 23.5924 25.7474 21.9155 25.2196C21.2702 25.0368 21.0536 25.3256 20.8475 25.6003C20.6354 25.883 20.4345 26.1508 19.7893 25.8747C17.6234 24.9629 19.3117 16.0941 22.2826 16.4342C23.2017 16.5381 23.604 17.1175 24.0757 17.7967C24.5472 18.4758 25.0881 19.2547 26.284 19.7583L27.574 20.3014Z"
      fill="#FF0000"
    />
  </svg>,
]
`;
